### ISSUE TITLE
Logstash queue parser fail with Logstash 7

### ISSUE THREAD

**Issue by ktibi** [influxdata/telegraf#7629]
Hello,

telegraf have a plugin for logstash.

The actual parser works for losgstash 6 but the format had change during the release 7.

Before 7 : 
```
      "queue": {
        "events": 103,
        "type": "persisted",
        "capacity": {
          "queue_size_in_bytes": 1872391,
          "page_capacity_in_bytes": 67108864,
          "max_queue_size_in_bytes": 1073741824,
          "max_unread_events": 0
        },
```
After 7 : 
```
      "queue" : {
        "type" : "persisted",
        "events_count" : 0,
        "queue_size_in_bytes" : 37677344,
        "max_queue_size_in_bytes" : 6442450944
      },
```
We need to change the structure of data in the plugin.

**Comment by danielnelson** [comment]
Would you be able to collect the output of a few curl commands, this will help us update the test cases:
```
http://127.0.0.1:9600/_node/stats/jvm
http://127.0.0.1:9600/_node/stats/process
http://127.0.0.1:9600/_node/stats/pipelines
http://127.0.0.1:9600/_node/stats/pipeline
```

**Comment by ktibi** [comment]
Hello @danielnelson 

https://gist.github.com/ktibi/7ca84e0d22ff053578b3881ce671da43

**Comment by olddanmer** [comment]
I've faced with the same issue. Telegraf's logstash plugin can't parse

```
      "queue" : {
        "type" : "persisted",
        "events_count" : 2729970,
        "queue_size_in_bytes" : 3265998654,
        "max_queue_size_in_bytes" : 26843545600
```

**Comment by kcy1019** [comment]
I am having the same issue as @olddanmer.

**Comment by ktibi** [comment]
@kcy1019  Do you have plan for work on it ?

**Comment by sjwang90** [comment]
Closed by #9080
